php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java

```diff
@@ -587,6 +654,25 @@ private static ModelElement getLastUse(NamespaceScope namespaceScope, boolean gr
         return offsetElement;
     }
 
+    private static int getFirstPHPTagPosition(PHPParseResult parserResult, NamespaceScope namespaceScope) {
```

Comment:
Nitpick: I think we use this notation for PHP:

```suggestion
    private static int getFirstPhpTagPosition(PHPParseResult parserResult, NamespaceScope namespaceScope) {
```

Reply:
Fixed.